script.js

```diff
@@ -0,0 +1,135 @@
+
+const searchButton = document.querySelector('#search-button');
+let searchBar = document.querySelector('#search-input');
+
+function generateCards(movieObject){
+    let star = document.createElement('span');
+    star.classList.add('star');
+    let starContent = document.createTextNode('⭐️ ');
+    star.appendChild(starContent);
+    // document.body.appendChild(star);
+
+
+    //create rating
+    let rating = document.createElement('span');
+    let ratingContent = document.createTextNode(movieObject.vote_average);
+    star.classList.add('rating');
+    star.appendChild(ratingContent);
+    // document.body.appendChild(rating);
+
+    //create average 
+    let averageContainer = document.createElement('div');
+    star.classList.add('average');
+    averageContainer.appendChild(star);
+    averageContainer.appendChild(rating);
+    // document.body.appendChild(averageContainer);
+
+    // create image 
+    let image = document.createElement('img');
+    let defaultImage = "https://motivatevalmorgan.com/wp-content/uploads/2016/06/default-movie.jpg"
+    image.src = movieObject.poster_path ? "https://image.tmdb.org/t/p/w342" + movieObject.poster_path : defaultImage
+    image.className = 'movie-poster'; 
+    image.alt = movieObject.original_title; 
+
+    //create name section
+    let name = document.createElement('div');
+    name.classList.add('name');
+    name.innerText = movieObject.original_title;
+    // document.body.insertBefore(name, averageContainer.nextSibling);
+
+    //create movie section
+    let movie = document.createElement('section');
+    name.classList.add('movie');
+    movie.appendChild(image);
+    movie.appendChild(averageContainer);
+    movie.appendChild(name);
+    document.body.appendChild(movie);
+    
+    return movie;
+
+}
+
+const  movieContainer = document.querySelector('#movie-grid') 
+// const loadMoreButton = document.querySelector('#load-more-button')
+
```

Comment:
best practice to declare all global variables at the top of the page
also remove all dead code/comments before submitting